tests/integrations/test_dsm.py

```diff
@@ -17,16 +18,23 @@ def setup_dsm_kafka(self):
         self.r = weblog.get("/dsm?integration=kafka")
 
     def test_dsm_kafka(self):
+        print(self.r)
+        print(self.r.content)
+        logger.info(f"exectute dsm tests {self.r}, {self.r.content}")
+        logger.info(f"exectute dsm tests {self.r.status_code}")
+        print(self.r.status_code)
         assert str(self.r.content, "UTF-8") == "ok"
 
+        consumer_hash = 5700134256996109392 if context.library == "nodejs" else 4463699290244539355
+        producer_hash = 13592025863925263556 if context.library == "nodejs" else 16645700936287432977
```

Comment:
Unclear what these hashes are, but why are these different just for Node.js? Seems like .NET and Java were using the same hash already so I would think there should be a way for Node.js to do that too? Or is there some context I'm missing here?

Reply:
This is the hashing we are using to identify consumers and producers. The other libraries are using FNV-1 hashing, we couldn't find a FNV-1 library for node, so we are using FNV-1a instead, which gives a different hash. Piotr said it's not an issue.